<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Servidor Gianni</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="InicioServidor">
     <attribute name="title">
      <string>Inicio Servidor</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar Servidor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TamNiveles">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad de niveles:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Cantidad De niveles:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>141</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Cambio de Tama√±o</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Consola</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>611</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Graficas</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>111</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar Matriz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Graficar Pila</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>320</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Arbol de Punteos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>320</y>
        <width>80</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Niveles</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menua">
    <property name="title">
     <string>&amp;Juego</string>
    </property>
    <addaction name="actionsalir"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca &amp;de</string>
    </property>
    <addaction name="actionDesarrollador"/>
    <addaction name="actionAplicacion"/>
   </widget>
   <widget class="QMenu" name="menuManuales">
    <property name="title">
     <string>&amp;Manuales</string>
    </property>
    <addaction name="actionManual_Tecnico"/>
    <addaction name="actionManual_de_Usuario"/>
   </widget>
   <addaction name="menua"/>
   <addaction name="menuAcerca_de"/>
   <addaction name="menuManuales"/>
  </widget>
  <action name="actionsalir">
   <property name="text">
    <string>&amp;salir</string>
   </property>
  </action>
  <action name="actionDesarrollador">
   <property name="text">
    <string>&amp;Desarrollador</string>
   </property>
  </action>
  <action name="actionAplicacion">
   <property name="text">
    <string>&amp;Aplicacion</string>
   </property>
  </action>
  <action name="actionManual_Tecnico">
   <property name="text">
    <string>Manual Tecnico</string>
   </property>
  </action>
  <action name="actionManual_de_Usuario">
   <property name="text">
    <string>Manual de Usuario</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
